vars/piperStageWrapper.groovy

```diff
@@ -78,9 +78,8 @@ private void executeStage(script, originalStage, stageName, config, utils, telem
         globalExtensions = fileExists(globalInterceptorFile)
         // Pre-defining the real originalStage in body variable, might be overwritten later if extensions exist
         def body = originalStage
-
         // First, check if a global extension exists via a dedicated repository
-        if (globalExtensions) {
+        if (!Boolean.valueOf(script.env.PIPER_DISABLE_EXTENSIONS) && globalExtensions) {
```

Comment:
Something like this: 

```suggestion
        if ((!script.env.PIPER_DISABLE_EXTENSIONS || !Boolean.valueOf(script.env.PIPER_DISABLE_EXTENSIONS)) && globalExtensions) {
```

Reply:
The tests  when ran individually, works, but running the whole class fails it. The condition check, in general works.

Reply:
Maybe a more general issue. I found something similar on Friday. Starting with commit 9ed39cfa3f05c8944bb08c6eb114c868506f54e3 `MavenExecuteStaticCodeChecksTest` runs fine when triggered in single mode, but fails when all tests are executed.